db/seeds/dev_seeds.rb

```diff
@@ -182,29 +180,32 @@
   end
   CommunicationLog.create!(person: listing.person,
                            match: match,
-                           created_by: delivery_method&.name.include?("autoemail") ? User.first : User.all.sample,
```

Comment:
If i'm reading this correctly, `delivery_method` was always going to be `autoemail`, so the ternary here wasn't doing anything?

Reply:
@exbinary sorry didn't mean to resolve

Reply:
I was doing a passthrough on all the ones that got created (simulating users having created some), and was thinking that some wouldn't be auto, but, maybe I'm wrong?